modules/cat/cat/main.nf

```diff
@@ -49,6 +49,7 @@ process CAT_CAT {
     """
 
     stub:
+    prefix   = task.ext.prefix ?: "${meta.id}${file_list[0].substring(file_list[0].lastIndexOf('.'))}"
```

Comment:
`file_list` needs to be declared here, otherwise instances without a `task.ext.prefix` declared in the config will throw an error
```suggestion
    def file_list = files_in.collect { it.toString() }
    prefix   = task.ext.prefix ?: "${meta.id}${file_list[0].substring(file_list[0].lastIndexOf('.'))}"
```